packages/import-maps-process/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@import-maps/process",
```

Comment:
what about calling this `@import-maps/resolve`

Reply:
will not be the full story but probably waaayy easier to understand - I would say sounds good 👍

Reply:
renamed to `@import-maps/resolve`